index.html

```diff
@@ -6,5 +6,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe 3</title>
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <img src="" alt="">
+      <div>
+        <ul>
+          <li><a href="#home">HOME</a></li>
```

Comment:
Perhaps better to just stick with regular case e.g. 'Home', 'Services'
And we can make it all upercase in the css styling later